dev-lang/perl/perl-5.32.1.recipe

```diff
@@ -123,7 +126,8 @@ INSTALL()
 	chmod a+x $binDir/{perl,perlthanks}
 
 	# copy script into site_perl that takes care of massaging @INC into what we need.
-	cp sitecustomize.pl $prefix/non-packaged/lib/perl5/site_perl/$perlShortVersion/
+	sed "s|/lib/|/lib$secondaryArchSubDir/|g" sitecustomize.pl > $prefix/non-packaged/$relativeLibDir/perl5/site_perl/$perlShortVersion/sitecustomize.pl
+	# TODO: maybe generate the script dynamically in the recipe (using recipe variables) instead of statically in the patchset
```

Comment:
Just to clarify what I meant here:

Currently, the file `sitecustomize.pl` is added to the sources via a patch in the patchset (and again modified with a later patch). We could instead add a template `sitecustomize.pl.in` in additional-files and fill out the template with sed replacements, similar to how `*.rdef.in` files are used.

It would also simplify the file a bit, because it would not need both `/boot/system/` and `/boot/home/config/` prefixed paths, but could instead just use `$prefix` (i.e. `/packages/perl-...`) to cater for both cases.